Formula/s/sentry-native.rb

```diff
@@ -0,0 +1,34 @@
+class SentryNative < Formula
+  desc "Sentry SDK for C, C++ and native applications"
+  homepage "https://docs.sentry.io/platforms/native/"
+  url "https://github.com/getsentry/sentry-native.git",
+    tag:      "0.7.20",
+    revision: "69569b5eca87b5c2afd1ce5a0df7b01ded85023b"
```

Comment:
```suggestion
  url "https://github.com/getsentry/sentry-native.git",
      tag:      "0.7.20",
      revision: "69569b5eca87b5c2afd1ce5a0df7b01ded85023b"
```
Can we use a tarball instead?

Reply:
In fact, there are tarballs for two newer releases, the latest (0.8.1) from last week. Is there a reason to start with this old version instead?

Reply:
I couldn't make it work with tarball, because it does not contain submodules files (external/crashpad).

Reply:
Submodules should be declared in your formula as resources. See the `osqp` formula for an example.

Reply:
Thanks, learned something :)